modules/copy-post.php

```diff
@@ -200,10 +200,21 @@ public function remove_post_format_template( $settings ) {
 	 * @return array Array with the results of each update action.
 	 */
 	protected function update_likes_sharing( $source_post, $target_post_id ) {
-		$likes          = get_post_meta( $source_post->ID, 'switch_like_status', true );
-		$sharing        = get_post_meta( $source_post->ID, 'sharing_disabled', false );
-		$likes_result   = update_post_meta( $target_post_id, 'switch_like_status', $likes );
-		$sharing_result = update_post_meta( $target_post_id, 'sharing_disabled', $sharing );
+		$likes   = get_post_meta( $source_post->ID, 'switch_like_status', true );
+		$sharing = get_post_meta( $source_post->ID, 'sharing_disabled', false );
+
+		if ( isset( $likes ) ) {
```

Comment:
I think `'' !== $test` would be better here. `get_post_meta` would return `''` if not present, and so `isset` is true for `$likes`. Reviewing the codebase, looks like `switch_like_status` should be either `0` or `1`, so need to be more exact. I wish we could use `empty` like the `sharing_disabled` option, but the `0` breaks that.

Reply:
Updated the check but still have a couple of potential concerns, more in the comment below.